### DISCUSSION TITLE
The ele res pseudo-mod implementation just doesn't work well

### DISCUSSION THREAD

**Discussion by DNekrassov** [Kvan7/Exiled-Exchange-2#125] [General]
In order for this feature to work at all, you have to log into the trade site in the in-built browser. But that means it will log you out of the trade site if you were looking for items in your default browser.   
Even when trying to access the hidden mods and do a search that way, it still uses the pseudo search function. Overall making the search process very tedious, and I'll probably revert to v0.1.0 for the time being. I think it's just better to wait for pseudo mod functionality to be added to the trade site by itself, or greatly simplify the pseudo mod functionality.   

Any thoughts about this?

**Comment by sownice** [comment]
Yes, “total resistance” is still missing in the Trade2 database and therefore the query fails. Is it possible to disable the checkbox for “total resistance” or instead of the total resistance checkbox, all resistances are queried individually in a new query? Otherwise the macro works so far under Linux for most other affixes, I haven't tried all of them either. First of all, thank you for your efforts.

**Reply by Kvan7** [reply]
Not quite sure what you mean for this, as far as I can tell `+ to all elemental resistances` is a mod, so that part is fine. In the general sense `total res` is not available which is the reason to have the sum part?

**Reply by sownice** [reply]
![image](https://github.com/user-attachments/assets/7171c78b-f8d5-44e0-a974-bc6a11a007fa)

**Reply by Kvan7** [reply]
Sorry the imgur link isn't loading for me, you _should_ be able to just paste an image into here and github will upload it automatically. 

I looked and it *appears* that I am calling to fire res, cold res, lightning res, and + to all ele res, So as far as I can tell it is called "total ele res" but I shouldnt be doing anything relating to "total to all res". but the image may provide better information

**Reply by sownice** [reply]
I’m really new to GitHub, so I hope you understand what I mean. Currently, Poe Trade2 does not have a 'Total' affix. My question is: can the GUI be changed to include options for Fire, Cold, and Lightning resistances while removing the 'Total Elemental Resistance' checkbox? Or would this require too much work before the trade site is fully ready?

**Reply by Kvan7** [reply]
All good! 

Yeah, the "total ele res" mod is what the old pseudo mod was called back in PoE1. I just haven't changed the name for my own simplicity's sake since they will probably add it back in, soon hopefully. I am basically just reusing bits and parts of the old code to hack together a solution to have "fake pseudo" mods. See #130, I am going to try to get an option to disable this within like an hour. but no garuntees on that since I am on a different computer than normal

**Reply by Kvan7** [reply]
This works and is now in [v0.1.3](https://github.com/Kvan7/Exiled-Exchange-2/releases/tag/v0.1.3)

**Comment by Kvan7** [comment]
You are able to use [this release](https://github.com/Kvan7/Exiled-Exchange-2/releases/tag/v0.1.2-no-pseudo) which doesn't include the pseudo stuff.

**Comment by Kvan7** [comment]
I may also try and do a hotfix for this in an hour or so

**Reply by Kvan7** [reply]
fix is live [v0.1.3](https://github.com/Kvan7/Exiled-Exchange-2/releases/tag/v0.1.3)

**Comment by belph93** [comment]
is this still not working ? im on ver 0.2.0
![image](https://github.com/user-attachments/assets/4f1c098b-798c-44fd-89ab-32e2d7ffc727)

**Reply by Kvan7** [reply]
Setting under price-check to turn it off. https://kvan7.github.io/Exiled-Exchange-2/complex-query.html

**Reply by belph93** [reply]
tyvm buddy
